src/components/logout/logout_container.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
```

Comment:
Why are you calling this file `logout_container` and other containers are just the index file of the folder? maybe we should rename this to be `/src/components/logout/index.js`